centos_ci.sh

```diff
@@ -224,6 +278,16 @@ else
     cd gopath/src/github.com/minishift/minishift
     make gen_adoc_tar IMAGE_UID=$(id -u)
     docs_tar_upload $PASS
+  elif [[ "$JOB_NAME" = "minishift-release" ]]; then
+    GIT_USER="minishift-bot"
+    prepare;
+    cd gopath/src/github.com/minishift/minishift
+    bump_to_release_version $GIT_USER;
+    make release
```

Comment:
I am a bit intrigued here. Do we not have to do so some error handling here? If `make release` fails, we should not call add_release_notes, right?